pkg/storage/stores/indexshipper/compactor/deletion/request_handler.go

```diff
@@ -174,11 +171,6 @@ func (dm *DeleteRequestHandler) cancelDeleteRequestHandler(w http.ResponseWriter
 		return
 	}
 
-	if deleteRequest.CreatedAt.Add(dm.deleteRequestCancelPeriod).Before(model.Now()) {
```

Comment:
I think the comment about 'Remove the cancellation period from the API' was probably originally here?

Reply:
It was just deleting the property on the type. This is where the check gets removed